### DISCUSSION TITLE
After updating my layout preferences in the zellij config file, is there a way to reload sessions to reflect these changes?

### DISCUSSION THREAD

**Discussion by Feel-ix-343** [zellij-org/zellij#3028] [Q&A]
Currently, I have to create a new session in order for changes to the default layout to be applied. Is there a better way to do this?

**Comment by adoreblvnk** [comment]
there isn't a way to reload changes, but you can attach to an existing session with a new layout via the `on_force_close` option [here](https://zellij.dev/documentation/options#on_force_close).

that said, Zellij should support reloading configs as per https://github.com/zellij-org/zellij/issues/1646
